frontend/src/redux/Pokemon/__tests__/normalizer.test.ts

```diff
@@ -0,0 +1,24 @@
+import { PokemonInterface } from 'components/Pokemon/Pokemon';
+import { normalize } from 'services/PokemonNormalizer';
+
+const pokemonArray = [
+  { id: 1, name: 'bulbasaur', height: 7, weight: 69 },
+  { id: 2, name: 'ivysaur', height: 10, weight: 130 },
+];
+
+describe('Pokemon normalizer', () => {
+  describe('empty array case', () => {
+    it('Should return an empty object', () => {
+      expect(normalize([])).toEqual({});
+    });
+  });
+
+  describe('filled array case', () => {
+    it('Should return a filled object', () => {
+      const pokemons = normalize(pokemonArray);
+      pokemonArray.forEach((pokemon: PokemonInterface) => {
+        expect(pokemons[pokemon.id]).toEqual(pokemon);
```

Comment:
Your test is too smart, and do not cover everything.
If the result of normalize was falsy and like below:
```
{
  1: { id: 1, name: 'bulbasaur', height: 7, weight: 69 },
  2: { id: 2, name: 'ivysaur', height: 10, weight: 130 },
  3: { id: 1, name: 'bulbasaur', height: 7, weight: 69 },
  4: { id: 2, name: 'ivysaur', height: 10, weight: 130 },
}
`
``
The test will pass.

A dumber solution could be setting a constant `normalizedPokemons`:
```
{
  1: { id: 1, name: 'bulbasaur', height: 7, weight: 69 },
  2: { id: 2, name: 'ivysaur', height: 10, weight: 130 },
  3: { id: 1, name: 'bulbasaur', height: 7, weight: 69 },
  4: { id: 2, name: 'ivysaur', height: 10, weight: 130 },
}
```
And only checking that the result equals `normalizedPokemons`